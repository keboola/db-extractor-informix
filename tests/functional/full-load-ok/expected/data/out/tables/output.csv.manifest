{
    "destination": "output",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "simple"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "simple"
        },
        {
            "key": "KBC.schema",
            "value": "%A"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "column_metadata": {
        "id": %A,
        "name": %A,
        "date": %A
    },
    "columns": [
        "id",
        "name",
        "date"
    ]
}
