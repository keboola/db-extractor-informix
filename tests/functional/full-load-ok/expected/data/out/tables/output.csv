"1","Jack Dawson","2020-10-03 01:02:34.000"
"2","Xander Thomas",""
"3","Jay Macdonald","2020-10-01 10:20:30.000"
